src/controllers/expenses.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+const expensesService = require('../services/expenses');
+
+const getAll = async(req, res) => {
+  res.send(await expensesService.getAll());
+};
+
+const getById = async(req, res) => {
+  const expense = await expensesService.getById(req.params.expenseId);
+
+  if (!expense) {
+    res.statusCode = 404;
+    res.end();
```

Comment:
```suggestion
    res.sendStatus(404);
```
or
```suggestion
    throw ApiError.NotFound();
```
check all other alike places